platform/ext/target/nordic_nrf/common/core/target_cfg.c

```diff
@@ -1083,6 +1083,26 @@ enum tfm_plat_err_t spu_periph_init_cfg(void)
 #endif
 	spu_peripheral_config_secure(uart_periph_start, SPU_LOCK_CONF_LOCKED);
 
+	/* Configure the CTRL-AP mailbox interface to be secure as it is used by the secure ADAC service */
+	spu_peripheral_config_secure(NRF_CTRLAP_S_BASE, SPU_LOCK_CONF_LOCKED);
+
+	/* Configure NRF_MEMCONF to be secure as it could otherwise be used to corrupt secure RAM. */
+	spu_peripheral_config_secure(NRF_MEMCONF_S_BASE, SPU_LOCK_CONF_LOCKED);
+
+	/* Configure trace to be secure, as the security implications of non-secure trace are not understood */
+	spu_peripheral_config_secure(NRF_TAD_S_BASE, SPU_LOCK_CONF_LOCKED);
+
+	/* Configure these secret HW features to be secure, as the security implications of them being non-secure are not understood */
```

Comment:
Didn't you mean to  say `That these addresses are just  missing in MDK therefore using the addressed hard coded to configure them`